packages/mgt/tsconfig.json

```diff
@@ -4,9 +4,11 @@
     "outDir": "dist/es6",
     "sourceRoot": "src",
     "rootDir": "src",
-    "composite": true
+    "composite": true,
+    "resolveJsonModule": true,
+    "esModuleInterop": true
   },
   "references": [{ "path": "../mgt-element" }],
-  "include": ["src"],
+  "include": ["src", "src/**/*.ts", "src/**/strings.ts"],
```

Comment:
This seems a little redundant. Will it work with just "src"?

Reply:
10b7c82 removed